clients/da-vinci-client/src/main/java/com/linkedin/davinci/stats/AggKafkaConsumerServiceStats.java

```diff
@@ -0,0 +1,120 @@
+package com.linkedin.davinci.stats;
+
+import com.linkedin.venice.exceptions.VeniceException;
+import com.linkedin.venice.meta.ReadOnlyStoreRepository;
+import com.linkedin.venice.stats.AbstractVeniceAggStoreStats;
+import com.linkedin.venice.stats.StatsSupplier;
+import com.linkedin.venice.utils.SystemTime;
+import io.tehuti.metrics.MetricsRepository;
+import java.util.function.LongSupplier;
+
+
+/**
+ * This class is an aggregate place that keeps stats objects for multiple stores and total stats for each region for
+ * AggKafkaConsumerService.
+ */
+public class AggKafkaConsumerServiceStats extends AbstractVeniceAggStoreStats<KafkaConsumerServiceStats> {
+  public AggKafkaConsumerServiceStats(
+      String regionName,
```

Comment:
😂 It's indeed confusing. I can add some comments to the javadoc.

Metrics from `KafkaConsumerServiceStats` today is per region/colo. Their metrics have the prefix "kafka_service_for_region1." [(code)](https://github.com/linkedin/venice/blob/main/clients/da-vinci-client/src/main/java/com/linkedin/davinci/stats/KafkaConsumerServiceStats.java#L38)

Now, since we have to add some store-level metrics, and I want to leverage `AbstractVeniceAggStoreStats`, I want `KafkaConsumerServiceStats` emits store-level metrics [(code)](https://github.com/linkedin/venice/pull/832/files#diff-4d6cbae558ac31fac337c6288ee5bb6f524ea066b3bbcb2eeada5ce83ce6779c), and `AggKafkaConsumerServiceStats` emits the total metrics for the given region, so that's why it takes a regionName.